### ISSUE TITLE
Validator for last step

### ISSUE THREAD

**Issue by chemsseddine** [r-bt/react-stepz#4]
Hello there, I am trying to set a validator for the last step but for some reason it's not working, is it an expected behavior ?

**Comment by r-bt** [comment]
No, it should be working for the last step. I'll take a look into it and get back to you
